torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -286,6 +287,75 @@ int64_t normalizeIndex(int64_t idx, int64_t list_size) {
 
 RegisterOperators reg(
     {Operator(
+         prim::profile,
+         [](const Node* node) -> Operation {
+           auto callback = node->cast<ProfileOp>()->getCallback();
+           return [callback](Stack& stack) {
+             callback(stack);
+             return 0;
+           };
+         },
+         aliasAnalysisSpecialCase()),
+     Operator(
+         prim::CudaFusionGroup,
+         [](const Node* node) -> Operation {
+#ifndef C10_MOBILE
```

Comment:
@csarofeen - I still see an ifdef here (and thank you for refactoring all other place!). Won't it be possible to just move the entire `RegisterOperators` call here to be linked into `libtorch_cuda.so`? (it can be a separate file or it can be in `cuda_graph_fuser.cpp`).